From 581499260c13076094916fd57d962098ac800eaf Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Tue, 17 Jun 2025 13:43:40 +0300
Subject: [PATCH] [SW-229758] gaudi2: bump svn to 11

Due to updates in Gaudi2E it is required to bump the SVN to 11.

Change-Id: I401f7b00a52df02e38b91bfd1e4b786a7f1235f6
Signed-off-by: Meron Cohen <mecohen@habana.ai>
(cherry picked from commit 1a94803a4e06f8f8f64f67baab3412d8dc149f30)
---

diff --git a/gaudi2/gaudi2_security.h b/gaudi2/gaudi2_security.h
index b3da78e..c3103a8 100644
--- a/gaudi2/gaudi2_security.h
+++ b/gaudi2/gaudi2_security.h
@@ -10,7 +10,7 @@
 
 #include <hl_sec_if.h>
 
-#define HL_SEC_FW_VERSION			10
+#define HL_SEC_FW_VERSION			11
 #define HL_SEC_KEY_INDEX			0x0
 
 #define HL_SDRAM_BASE				0x11000000
